Fixture Factories

pabloelcolombiano opened the pull request:
A tested application requires to write test cases quickly and numerously with taylor-made data
for your test. CakePHP fixtures being static, they do not meet this requirement.

The present package proposes an dynamic solution to create test fixtures.
https://github.com/pakacuda/cakephp-fixture-factories

dereuromark commented:
Isnt the name CakephpFixtureFactories ?
Please also see the other guidelines, e.g. alphabetical etc.

pabloelcolombiano commented:
Thanks for the review Mark. 

I have added an MIT licence in the package Readme.md, and renamed the plugin in the present repo as "CakePHP Fixture Factories plugin".

Let me know if anything else is missing from your poitn of view.

dereuromark commented:
The license info in the readme is neither relevant nor legally acceptable enough.
You need to follow the guidelines outlined in this repo as well as how all other plugins are also handling this.

dereuromark commented on README.md:
```diff
@@ -413,6 +413,7 @@ Additional lists you might find useful:
 
 - [CakePHP Codeception module](https://github.com/cakephp/codeception) - The official CakePHP integration with [Codeception](https://codeception.com).
 - [CakePHP CodeSniffer rules](https://github.com/cakephp/cakephp-codesniffer) - The official CakePHP CS rules.
+- :strawberry: [CakePHP Fixture Factories plugin](https://github.com/pakacuda/cakephp-fixture-factories) - Create your fixtures dynamically on a test basis, accelerate the writing and maintainance of your tests.
```
That is not the name, we need the 100% match here, I posted it for the reason to clarify. You used a different name now again after fixing the initially wrong one.

pabloelcolombiano commented:
Is there anything left missing from my side for a merge? Just wondering, but of course take your time if review on your side is needed!
